Examples/Widgets/CRM/src/components/VisitBadge.scss

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+ @import "../assets/styles/variables";
+
+@import "./TableStyles";
+
+:host {
+  width: 100vw;
+}
+
+.mock-badge {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  color: var(--md-primary-text-color, #777);
+  background-color: var(--md-primary-bg-color, #fff);
+  border-radius: 1.5rem;
+  padding: 0.25rem 1rem 0.25rem 0;
+  height: 1rem;
+  white-space: nowrap;
+  font-size: 0.625rem;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.16), 0px 0px 1px rgba(0, 0, 0, 0.18);
+  z-index: 1;
+  cursor: pointer;
+  
+  &:hover {
+    background-color: var(--badge-cyan-bg-color,#e6fbff);
```

Comment:
a lot of hardcoded colors i see

Reply:
So my thinking was reach for the theme-responsive token that exists and provides the hues from the design . . . are you referring more to the hex fallback? I can move that to variables.scss